src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy

```diff
@@ -113,7 +113,10 @@ class SimpleAndroidApp {
                 }
             """.stripIndent()
 
-        file("${library}/build.gradle") << subprojectConfiguration("com.android.library") << activityDependency()
+        file("${library}/build.gradle") << subprojectConfiguration("com.android.library") << activityDependency() <<
+            """
+                tasks['assemble'].dependsOn 'createFullJarDebug', 'createFullJarRelease'
```

Comment:
instead of adding a task dependency, can we add `createFullJarDebug` and createFullJarRelease` on the command line?
does adding `createFullJar` to the command line arguments achieve the same thing?

Reply:
done, it might make more sense indeed